libs/blocks/chart/chart.js

```diff
@@ -66,19 +66,32 @@ const barTooltipFormatter = ({
   `${seriesName}<br />${marker} ${value[x[0]]}${unit} ${name}<i class="tooltip_icon"></i>`
 );
 
-const barSeriesOptions = (seriesData, colors, unit) => (
-  seriesData.map((value, index) => ({
+const barSeriesOptions = (seriesData, colors, size, unit) => {
+  const isLarge = size === LARGE;
+
+  return seriesData.map((value, index) => ({
     type: 'bar',
     label: {
       show: true,
       formatter: `{@[${index + 1}]}${unit}`,
+      position: 'right',
+      textBorderColor: '#000',
+      distance: 8,
+      fontSize: isLarge ? 16 : 14,
     },
     showBackground: true,
-    backgroundStyle: { color: colors[index] },
-  }))
-);
+    backgroundStyle: {
+      color: colors[index],
+      borderRadius: 3,
+      opacity: 0.35,
+    },
+    itemStyle: { borderRadius: 3 },
+    barCategoryGap: 0,
+    barGap: '33.3%',
+  }));
+};
 
-export const getChartOptions = (chartType, dataset, colors, unit = '') => {
+export const getChartOptions = (chartType, dataset, colors, size, unit = '') => {
```

Comment:
I have the same question here about code comments

Reply:
One option is passing in an object which makes it somewhat self documenting.
```suggestion
export const getChartOptions = ({chartType, dataset, colors, size, unit = ''}) => {
```

Since all milo/hlx files are loaded without any pre-processing, it's best to keep the amount of comments minimal.

Reply:
@JasonHowellSlavin Do you have any response here?